x/oracle/types/oracle.go

```diff
@@ -94,6 +94,10 @@ func (m *OracleRegistration) ValidateBasic() error {
 		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "OracleCommissionMaxRate must be between 0 and 1")
 	}
 
+	if m.OracleCommissionRate.GT(m.OracleCommissionMaxRate) {
```

Comment:
I think we can change like below.(line: 89)

```go
if m.OracleCommissionRate.LT(sdk.ZeroDec()) || m.OracleCommissionRate.GT(m.OracleCommissionMaxRate) {
	return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "oracleCommissionRate must be between 0 and 1")
}
```

How about it?

Reply:
Got it. It looks better. I'll change it